### PR TITLE
zh,en;replace `kubectl edit` to `kubectl patch`

### LINKED ISSUE #1048: Use `kubectl patch` to replace `kubectl edit` for user experience improvement
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by better0332**
### What is changed, added, or deleted? (Required)
replace `kubectl edit` to `kubectl patch`

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

Fix #1048

### CODE REVIEW

File: en/deploy-on-gcp-gke.md [hunk 1]
```diff
@@ -254,7 +254,7 @@ You can access the `${grafana-lb}:3000` address using your web browser to view m
 
 ## Upgrade
 
-To upgrade the TiDB cluster, edit the `spec.version` by executing `kubectl edit tc basic -n tidb-cluster`.
+To upgrade the TiDB cluster, executing command `kubectl patch tc basic -n tidb-cluster --type merge -p '{"spec":{"version":"${version}"}}`.
```
Comment (ran-huang) [inline en/deploy-on-gcp-gke.md:257]:
````suggestion
To upgrade the TiDB cluster, execute the following command:

{{< copyable "shell-regular" >}}

```shell
kubectl patch tc basic -n tidb-cluster --type merge -p '{"spec":{"version":"${version}"}}`.
```
````

File: en/deploy-on-azure-aks.md [hunk 2]
```diff
@@ -391,7 +391,7 @@ See [Access TiDB Dashboard](access-dashboard.md) for instructions about how to s
 
 ## Upgrade
 
-To upgrade the TiDB cluster, edit `spec.version` by running the `kubectl edit tc basic -n tidb-cluster` command.
+To upgrade the TiDB cluster, executing command `kubectl patch tc basic -n tidb-cluster --type merge -p '{"spec":{"version":"${version}"}}`.
```
Comment (ran-huang) [inline en/deploy-on-azure-aks.md:394]:
````suggestion
To upgrade the TiDB cluster, execute the following command:

{{< copyable "shell-regular" >}}

```shell
kubectl patch tc basic -n tidb-cluster --type merge -p '{"spec":{"version":"${version}"}}`.
```
````

File: en/deploy-on-aws-eks.md [hunk 3]
```diff
@@ -512,7 +512,7 @@ See [Access TiDB Dashboard](access-dashboard.md) for instructions about how to s
 
 ## Upgrade
 
-To upgrade the TiDB cluster, edit the `spec.version` by executing `kubectl edit tc basic -n tidb-cluster`.
+To upgrade the TiDB cluster, executing command `kubectl patch tc basic -n tidb-cluster --type merge -p '{"spec":{"version":"${version}"}}`.
```
Comment (ran-huang) [inline en/deploy-on-aws-eks.md:515]:
````suggestion
To upgrade the TiDB cluster, execute the following command:

{{< copyable "shell-regular" >}}

```shell
kubectl patch tc basic -n tidb-cluster --type merge -p '{"spec":{"version":"${version}"}}`.
```
````

**Comment by DanielZhangQD** [comment]
@KanShiori PTAL.

**Review by ran-huang — APPROVED** [review]

**Review by KanShiori — APPROVED** [review]
LGTM

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**